src/vkbd/usb-gadget.h

```diff
@@ -0,0 +1,152 @@
+#ifndef _USBGADGET_H_
+#define _USBGADGET_H_
+#define _USBGADGET_H_
+
+#define HID_ALT_GR 0x40
+#define HID_CTRL 0x1
+#define HID_SHIFT 0x2
+#define HID_SUPER 0x8
+#define HID_ALT 0x4
+
+static const uint16_t evdev_to_hid[276] = {
```

Comment:
You can omit the array size in a literal, so this can just be `evdev_to_hid[]`

Reply:
Done.

Reply:
OK